protocol/contracts/libraries/Token/LibTransfer.sol

```diff
@@ -110,20 +110,4 @@ library LibTransfer {
             token.burn(burnt);
         }
     }
-
-
-    // ask publius, address(this) would refer to the beanstalk addy right
```

Comment:
I also removed this function in my branch, but I believe I removed the other one haha, lets check this when we re-merge